checker/jdk/nullness/src/java/util/concurrent/ConcurrentSkipListSet.java

```diff
@@ -225,7 +225,7 @@ public ConcurrentSkipListSet(SortedSet<E> s) {
      *         compared with the elements currently in this set
      * @throws NullPointerException if the specified element is null
      */
-    @Pure public boolean contains(Object o) {
+    @Pure public boolean contains(@Nullable Object o) {
```

Comment:
Similarly, I don't think we want `@Nullable` on `contains` and `remove` in `ConcurrentSkipListSet`.